lib/compilers/carbon.ts

```diff
@@ -65,41 +67,33 @@ export class CarbonCompiler extends BaseCompiler {
         );
     }
 
-    override async afterCompilation(
-        result,
-        doExecute,
-        key,
-        executeParameters,
-        tools,
-        backendOptions,
-        filters,
-        options,
-        optOutput,
-        customBuildPath,
-    ) {
-        result = await super.afterCompilation(
-            result,
-            doExecute,
-            key,
-            executeParameters,
-            tools,
-            backendOptions,
-            filters,
-            options,
-            optOutput,
-            customBuildPath,
-        );
+    lastLine(lines?: ResultLine[]): string {
+        if (!lines) return '';
```

Comment:
Maybe `if(!lines || lines.length === 0)`?

Reply:
I was trying `?` thing but obviously we still haven't fixed that either. I'm not sure the type system is clever enough to "know" that `length === 0` means that `.at(-1)` won't return `undefined`, so it'd need a cast?

Reply:
the `as` worked OK though! Will commit and we can work on it :)